db/migrate/20170717105219_copy_confirmed_at_from_enterprises_to_spree_users.rb

```diff
@@ -0,0 +1,5 @@
+class CopyConfirmedAtFromEnterprisesToSpreeUsers < ActiveRecord::Migration
+  def up
+    execute "UPDATE spree_users SET confirmed_at = enterprises.confirmed_at FROM enterprises WHERE spree_users.email = enterprises.email AND enterprises.confirmed_at IS NOT NULL"
+  end
+end
```

Comment:
@ltrls I'm thinking maybe we should include this update query at the bottom of the `up` method of the `AddConfirmableToUser` migration instead of independently.